### DISCUSSION TITLE
How to filter posts where `translations` array is not empty?

### DISCUSSION THREAD

**Discussion by meerzulee** [directus/directus#9872] [Archive]
### For example this returns items and "translations" array with filtered language: 
`/items/publications?deep[translations][_filter][languages_code][_eq]=en-US` 

![image](https://user-images.githubusercontent.com/17146298/142147003-b85af5d2-179c-4bc9-982b-9eec345c7b1f.png)

I need to get all posts where "translations" array is not empty. 
I tried to add `filter[translations][_nempty]=true` and  `_and` to `deep` filter  but it returns error.

**How to filter posts where `translations` array is not empty?**

**Comment by rijkvanzanten** [comment]
There's currently no filters that can check for the number of related items in one to many relationships
